spec/controllers/settings/profiles_controller_spec.rb

```diff
@@ -23,5 +23,10 @@
       expect(account.reload.display_name).to eq 'New name'
       expect(response).to redirect_to(settings_profile_path)
     end
+
+    it 'renders :show if parameter is missing' do
+      put :update
+      expect(response).to render_template :show
```

Comment:
> When the params/route is valid - but the data included in the update is invalid - I'd expect a 200 response with a rendered page that includes a description of the data validation errors.

Then why shouldn't it return 400 for invalid parameters while parameter missing does? I decided to return 200 for parameter missing because it does so for invalid parameters, which should have 400 if we respect RFC.